backend/aka/tests/test_errors.py

```diff
@@ -1,6 +1,7 @@
-from django.test import SimpleTestCase
 import json
 
+from django.test import SimpleTestCase
+
 
 class ErrorsIntegrityTestCase(SimpleTestCase):
```

Comment:
Not that descriptive of what you are testing. Better name would be ErrorMessagesTestCase with some doc strings.